Node crashes when providing a peers.json file

**Describe the bug**
When providing a peers.json file to recover from a failure, I get a runtime error.
I was juste following the documentation: https://dkron.io/usage/recovery/#manual-recovery-using-peersjson

**To Reproduce**
This is the peers.json file I am using:
```
[
 {"id":"i-0fb7e688e9dc2b55f","address":"10.32.5.138:6868"},
 {"id":"i-09accd4e7006e30ae","address":"10.32.4.170:6868"}
]
```
When I start the machine, it crashes whith this stack:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x68 pc=0x2056b31]

goroutine 1 [running]:
github.com/distribworks/dkron/v3/dkron.(*dkronFSM).applySetJob(0xc000ddf900, 0xc000e78241, 0x218, 0x218, 0x1, 0x2910b60)
	/dkron/dkron/fsm.go:82 +0xb1
github.com/distribworks/dkron/v3/dkron.(*dkronFSM).Apply(0xc000ddf900, 0xc000dc5e00, 0xc000dc5e00, 0x0)
	/dkron/dkron/fsm.go:59 +0x1e1
github.com/hashicorp/raft.RecoverCluster(0xc00103f5c8, 0x3541620, 0xc000ddf900, 0x356a5a0, 0xc000e99140, 0x3556aa0, 0xc0011b9280, 0x3543460, 0xc0011d08d0, 0x357d2e0, ...)
	/go/pkg/mod/github.com/hashicorp/raft@v1.1.2/api.go:351 +0x4d3
github.com/distribworks/dkron/v3/dkron.(*Agent).setupRaft(0xc000d72000, 0x354eea0, 0xc0011b91e0)
	/dkron/dkron/agent.go:331 +0xef5
github.com/distribworks/dkron/v3/dkron.(*Agent).StartServer(0xc000d72000)
	/dkron/dkron/agent.go:551 +0x5c1
github.com/distribworks/dkron/v3/dkron.(*Agent).Start(0xc000d72000, 0xc0009dfcc8, 0x1)
	/dkron/dkron/agent.go:185 +0x6d2
github.com/distribworks/dkron/v3/cmd.agentRun(0x4b079e0, 0x0, 0x0, 0x0, 0x0)
	/dkron/cmd/agent.go:64 +0x174
github.com/distribworks/dkron/v3/cmd.glob..func1(0x4ab4440, 0x4b079e0, 0x0, 0x0, 0x0, 0x0)
	/dkron/cmd/agent.go:38 +0x3f
github.com/spf13/cobra.(*Command).execute(0x4ab4440, 0x4b079e0, 0x0, 0x0, 0x4ab4440, 0x4b079e0)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0x4ab46e0, 0x442b1a, 0x4a39b20, 0xc000062778)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/distribworks/dkron/v3/cmd.Execute()
	/dkron/cmd/dkron.go:32 +0x31
main.main()
	/dkron/main.go:9 +0x20
```

**Expected behavior**
The node should not crash.

**Screenshots**
N/A

** Specifications:**
 - OS: AWS EC linux (latest)
 - Version [e.g. 3.0.2]

**Additional context**

--- a/dkron/agent.go
+++ b/dkron/agent.go
@@ -327,7 +327,11 @@ func (a *Agent) setupRaft() error {
 			if err != nil {
 				return fmt.Errorf("recovery failed to parse peers.json: %v", err)
 			}
-			tmpFsm := newFSM(nil, nil)
+			store, err := NewStore()
+			if err != nil {
+				log.WithError(err).Fatal("dkron: Error initializing store")
+                        }
+			tmpFsm := newFSM(store, nil)
 			if err := raft.RecoverCluster(config, tmpFsm,
 				logStore, stableStore, snapshots, transport, configuration); err != nil {
 				return fmt.Errorf("recovery failed: %v", err)

